tests/unit/models/gpflow/test_models.py

```diff
@@ -1618,6 +1619,60 @@ def test_sparse_variational_pairwise_covariance_for_non_whitened(
     np.testing.assert_allclose(expected_covariance, actual_covariance, atol=1e-4)
 
 
+class DummyInducingPointSelector(InducingPointSelector[GPflowPredictor]):
+    def __init__(self, new_inducing_points: TensorType, recalc_every_model_update: bool = True):
+        super().__init__(recalc_every_model_update)
+        self._new_inducing_points = new_inducing_points
+
+    def _recalculate_inducing_points(
+        self, M: int, model: ProbabilisticModelType, dataset: Dataset
+    ) -> TensorType:
+        return self._new_inducing_points
+
+
+@random_seed
+@pytest.mark.parametrize("whiten", [False, True])
+def test_sparse_variational_inducing_updates_preserves_posterior(
+    whiten: bool,
+) -> None:
+    from gpflow.config import Config, as_context
+
+    from trieste.utils import DEFAULTS
+
+    default_jitter = 0.0
+    DEFAULTS.JITTER = default_jitter
```

Comment:
@uri-granta this line works fine on my machine but fails otherwise... any clue?

Reply:
That's actually encouraging! (we shouldn't be able to override the DEFAULTS accidentally) See my other comment for how to temporarily change them.